server/server.js

```diff
@@ -9,8 +9,15 @@ const config = require('./config');
 const httpsServer = require('./https_server/https_server');
 const redirectServer = require('./https_server/redirect_server');
 
-var app = express();
+const app = express();
+const bodyParser = require('body-parser');
+// router must be after body-parser
+const router = require('./router/main.js')(app);
+
 app.use(express.static(path.join(__dirname, '../client')));
+app.use(bodyParser.urlencoded({
+  extended: true}));
```

Comment:
Isn't it right to enter a line like below?
app.use(bodyParser.urlencoded({
  extended: true
}));